source/install/virtual-machine.rst

```diff
@@ -17,21 +17,113 @@ The core distribution builds contain the following plugins:
+
+5. Launch the virtual machine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When prompted, select the user ``qiime2``, and enter the password ``qiime2``.
+
+.. image:: images/vbox-install/vbox-04.jpg
+
+Once logged in you should see a screen like this:
+
+.. image:: images/vbox-install/vbox-05.jpg
+
+Success!
+
+6. Clean up
+~~~~~~~~~~~
+
+The import process creates a copy of the files from the directory that you unzipped in step 3 above. If you would like to reclaim that disk space, feel free to delete ``qiimeX_Y_Z-build_number.zip``, and the unzipped folder from that file.
+
+Optional: Shared Folders and Copy-and-Paste
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Install Guest Additions
+..........................
+
+After clicking on the CD icon in the lower left, click on ``Run Software`` in the upper right.
+
+.. image:: images/vbox-guest/vbox-01.jpg
+
+Enter the password (``qiime2``) when prompted.
+
+.. image:: images/vbox-guest/vbox-02.jpg
+
+When installation is done, your screen should look like the following. Follow the prompt and press ``Return``
+
+.. image:: images/vbox-guest/vbox-03.jpg
+
+2. Set up a shared folder
+.........................
+
+In the VirtualBox menu bar on your host computer, select ``Devices -> Shared Folders -> Shared Folders Settings`` (note, this might be slightly different on Windows or Linux systems, please check the official `VirtualBox Docs`_). Click on the green plus sign/folder on the right side of the window, you should see a new screen prompting you for info:
+
+.. image:: images/vbox-guest/vbox-04.jpg
+
+Click on the ``Folder Path`` input, and using the file browser (by selecting the option ``Other...``), select the folder you want to share with the VM. This example uses the Desktop (e.g. ``/Users/matthew/Desktop`` on a Mac). Make sure to select ``Auto-mount`` and ``Make Permanent``, if you don't want to repeat these steps every time you launch the VM. Also note, the ``Folder Name`` option (which VirtualBox will auto-suggest a value for) is the name of the mounted folder inside of your VM.  VirtualBox will always prepend the prefix ``sf_`` to that value (``sf`` stands for ``shared folder``). So in this case, the folder name will be ``sf_Desktop``. The full path when mounted is ``/media/sf_Desktop``.
+
+.. image:: images/vbox-guest/vbox-05.jpg
+
+Click 'OK' on both dialogs to save your changes.
+
+3. Reboot
+.........
+
+Click the gear in the upper right of the screen, then ``Shut Down...``, then ``Restart``.
+
+.. image:: images/vbox-guest/vbox-06.jpg
+
+4. Enjoy!
+.........
+
+Once the VM has rebooted and you have logged in, you should see your shared folder in the file browser (the icon on the left bar that looks like a file cabinet):
+
+.. image:: images/vbox-guest/vbox-07.jpg
+
+The files there are the files on the Mac's desktop! These files can also access the mounted folder via the Command Line at ``/media/sf_Desktop``:
+
+.. image:: images/vbox-guest/vbox-08.jpg
```

Comment:
Can you describe how to create a symbolic link to the shared folder? The QIIME 1 VB had a symlink in Desktop and users seemed to like that. This could probably just be a `.. tip::` so you don't have to create screenshots (up to you).